core-bundle/src/Resources/contao/themes/flexible/install.css

```diff
@@ -137,6 +142,7 @@ p {
 .tl_info {
 	color:#006494;
 	background:url("icons/show.svg") no-repeat 1px 3px;
+	background:url("icons/show.svg") no-repeat 1px 3px;
```

Comment:
Typo?

Reply:
I wanted to make extra sure this CSS is in there, so I doubled the line.


typo for sure  :smile: